### ISSUE TITLE
i18n to RECAPTCHA_ERROR_CODES

### ISSUE THREAD

**Issue by DmonteiroD** [lepture/flask-wtf#198]
It will be helpful if this messages could be translated without modify locally this file:

flask_wtf/recaptcha/validators.py

RECAPTCHA_ERROR_CODES = {
    'missing-input-secret': 'The secret parameter is missing.',
    'invalid-input-secret': 'The secret parameter is invalid or malformed.',
    'missing-input-response': 'The response parameter is missing.',
    'invalid-input-response': 'The response parameter is invalid or malformed.'
}

**Comment by sobolevn** [comment]
I can make a PR, just fixed it for myself.

**Comment by davidism** [comment]
@sobolevn that would be great.  I was trying to, but I *really* don't get i18n I guess. :confounded:

**Comment by sobolevn** [comment]
Any plans to support https://www.transifex.com/ ?

**Comment by davidism** [comment]
I don't think we have the resources for transifex/crowdin/etc.  Or does support not involve buying a plan?  That would be a separate issue though.

**Comment by tristanfisher** [comment]
I'd imagine this would just be slurping in some localized JSON blob in a constructor (e.g. validators_en.json, validators_ru.json) -- 3rd party stuff seems like overkill.

**Comment by sobolevn** [comment]
I have done it using `babel`. I have used the same approach as [`wtform`](https://github.com/wtforms/wtforms/tree/master/wtforms/locale). Is it fine by everyone?

I guess I will be able to submit it for review pretty soon.

**Comment by sobolevn** [comment]
Sorry, forgot to mention this issue: https://github.com/lepture/flask-wtf/pull/253

**Comment by sobolevn** [comment]
@davidism Really looking forward your feedback!

**Comment by PanderMusubi** [comment]
An approach such as https://github.com/wtforms/wtforms/tree/master/src/wtforms/locale is welcome.

**Comment by PanderMusubi** [comment]
> I don't think we have the resources for transifex/crowdin/etc. Or does support not involve buying a plan? That would be a separate issue though.

Weblate is free for FOSS. See the Libre plan at the bottom of https://weblate.org/en/hosting/
